### PR TITLE
Update mainWindow.ui

### PR DESCRIPTION
**Pull request by soble**
Add shortcut to export cloud 2 line csv summary
> Tool > Batch export > Export cloud info

### CODE REVIEW

File: qCC/ui_templates/mainWindow.ui [hunk 1]
```diff
@@ -39,7 +39,7 @@
      <x>0</x>
      <y>0</y>
      <width>1024</width>
-     <height>31</height>
+     <height>21</height>
```
Comment (dgirardeau) [inline qCC/ui_templates/mainWindow.ui:42]: Can you revert this change maybe? (or is intentional?)

**Comment by soble** [comment]
Deleted this merge request and requested another with a different patch.
